openpype/lib/path_templates.py

```diff
@@ -280,8 +280,21 @@ def templates(self):
     def objected_templates(self):
         return self._objected_templates
 
-    @classmethod
-    def create_ojected_templates(cls, templates):
+    def _create_template_object(self, template):
+        """Create template object from a template string.
+
+        Separated into method to give option change class of templates.
+
+        Args:
+            template (str): Template string.
+
+        Returns:
+            StringTemplate: Object of template.
+        """
+
+        return StringTemplate(template)
+
+    def create_ojected_templates(self, templates):
```

Comment:
Since we're changing this line of code anyway, should we fix the typo?

```suggestion
    def create_objected_templates(self, templates):
```
Might need to refactor other areas of the code however.